src/Resources/skeleton/command/Command.php.txt

```diff
@@ -22,6 +22,26 @@ class {{ command_class_name }} extends Command
         ;
     }
 
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        //Optional.
+        //If no initialization is required either leave this empty or remove it.
+
+        //This method is executed before the interact() and the execute() methods.
+        //Its main purpose is to initialize variables used in the rest of the command methods.
+    }
```

Comment:
Hmm. I'd rather *not* have this one. I've actually never used it before. And I think `__construct()` might now be a better place.

Reply:
Depending on what needs to be initialized you are right `__construct` is the place to go, unless you need the user provided arguments for initialization. In this case often execute is used directly which clutters the actual task that's to be done a bit. But of course it's a matter of taste.